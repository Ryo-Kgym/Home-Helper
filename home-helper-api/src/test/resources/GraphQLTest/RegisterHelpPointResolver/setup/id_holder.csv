id_type,current_id
HELP_POINT_EARNED_ACHIEVEMENT_ID,1
HELP_POINT_EARNED_DETAIL_ID,2
