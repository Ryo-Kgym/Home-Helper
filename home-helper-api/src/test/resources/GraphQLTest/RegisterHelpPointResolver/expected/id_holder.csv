id_type,current_id
HELP_POINT_EARNED_ACHIEVEMENT_ID,2
HELP_POINT_EARNED_DETAIL_ID,5
